package.json

```diff
@@ -9,10 +9,17 @@
   },
   "devDependencies": {
     "fs-extra": "10.0.0",
-    "markdown-table": "3.0.2"
+    "husky": "7.0.0",
+    "markdown-table": "3.0.2",
+    "prettier": "2.5.1"
```

Comment:
@viceice I'm not sure how to use/install `lint-staged`, feel free to push to this PR to add that. 😉

Reply:
Will do this in separate pr